http.go

```diff
@@ -0,0 +1,68 @@
+package discordgo
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+)
+
+type WebhookServer struct {
+	sess      *Session
```

Comment:
maybe just embed the session like that?
```suggestion
	*Session
```